pkg/rbd/rbd_util.go

```diff
@@ -163,6 +163,30 @@ func rbdStatus(pOpts *rbdVolume, cr *util.Credentials) (bool, string, error) {
 	return false, output, nil
 }
 
+// rbdManagerTaskDelete adds a ceph manager task to delete an rbd image, thus deleting
+// it asynchronously. If command is not found returns a bool set to false
+func rbdManagerTaskDeleteImage(pOpts *rbdVolume, cr *util.Credentials) (bool, error) {
+	var output []byte
+
+	args := []string{"rbd", "task", "add", "remove",
+		pOpts.Pool + "/" + pOpts.RbdImageName,
+		"--id", cr.ID,
+		"--keyfile=" + cr.KeyFile,
+		"-m", pOpts.Monitors,
+	}
+
+	output, err := execCommand("ceph", args)
+	if err != nil {
+		if strings.Contains(string(output), "no valid command found") &&
+			strings.Contains(string(output), "Error EINVAL: invalid command") {
+			klog.Infof("cluster with cluster ID (%s) does not support Ceph manager based rbd image deletion", pOpts.ClusterID)
```

Comment:
log this as a warning

Reply:
I think the first check for "no valid command found" will be sufficient for this isn't it?

Reply:
what is the minimum supported ceph version for this command?
please add a comment

Reply:
I am going to do the following, let me know if it works for you.
- Retain info, as this is just informational and not a warning message (using info here was by purpose)
- Retain 2 checks as I prefer to be sure, and use more than one message output available to detect the same
- Add the minimum supported version as 14.3, but it may so happen that the feature maybe backported to mimic as well, at which point we may need to revisit this, and so will add a comment in the code around this

Reply:
>     * Add the minimum supported version as 14.3, but it may so happen that the feature maybe backported to mimic as well, at which point we may need to revisit this, and so will add a comment in the code around this

It would be v14.2.3 -- and you are correct that it will be backported to mimic as well.